src/pages/HttpBin/index.js

```diff
@@ -15,7 +15,7 @@ import axios from 'axios';
 import FormInput from '../../components/FormInput';
 import SelectFormInput from '../../components/SelectFormInput';
 
-import exampleJson from '../../data/httpbin_example.json';
+import { httpbin } from '../../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file